internal/gtoc/header.go

```diff
@@ -29,7 +29,8 @@ func (c *Converter) newHeader(inv *bill.Invoice) error {
 		notes := make([]*document.Note, 0, len(inv.Notes))
 		for _, n := range inv.Notes {
 			notes = append(notes, &document.Note{
-				Content: n.Text,
+				Content:     n.Text,
+				SubjectCode: n.Code,
```

Comment:
I would add a nil check here, as it is possible for a note not to include Code.

Reply:
@apardods Hey, thx for having a look. I just checked the code, and both entities `cbc.Note` and `document.Note` have fields `Code` and `SubjectCode` respectively declared as just a `string`, so they can't be nil.

I tested note without a Code, it works as an empty string and doesn't end up in the final xml

Reply:
Perfect! As long it an invoice with no code does not show the empty field great